prow/config/config.go

```diff
@@ -153,6 +154,10 @@ type ProwConfig struct {
 	// DefaultJobTimeout this is default deadline for prow jobs. This value is used when
 	// no timeout is configured at the job level. This value is set to 24 hours.
 	DefaultJobTimeout *metav1.Duration `json:"default_job_timeout,omitempty"`
+
+	// ManagedWebhooks contains information about all github repositories and organizations which are using
+	// non-global Hmac token.
+	ManagedWebhooks ManagedWebhooks `json:"managed_hmac_entity,omitempty"`
```

Comment:
```suggestion
	ManagedWebhooks ManagedWebhooks `json:"managed_webhooks,omitempty"`
```

Reply:
Otherwise LGTM.

Reply:
I am sure I made this change! not sure how it got reverted.